pkg/controllers/resources/priorityclasses/translate.go

```diff
@@ -18,18 +18,42 @@ func (s *priorityClassSyncer) translate(ctx *synccontext.SyncContext, vObj clien
 	return priorityClass
 }
 
-func (s *priorityClassSyncer) translateUpdate(pObj, vObj, sourceObject, targetObject *schedulingv1.PriorityClass) {
+func (s *priorityClassSyncer) translateFromHost(ctx *synccontext.SyncContext, pObj client.Object) *schedulingv1.PriorityClass {
+	// translate the priority class
+	priorityClass := translate.VirtualMetadata(pObj.(*schedulingv1.PriorityClass), s.HostToVirtual(ctx, types.NamespacedName{Name: pObj.GetName(), Namespace: pObj.GetNamespace()}, pObj))
+	priorityClass.GlobalDefault = false
+	if priorityClass.Value > 1000000000 {
+		priorityClass.Value = 1000000000
+	}
```

Comment:
Why are we doing these adjustments here, I know they are needed when going from virtual -> host, but in this case we could just leave them be, no?